app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/controllers/TermsOfUseFailedJourneyController.scala

```diff
@@ -170,7 +200,89 @@ class TermsOfUseFailedJourneyController @Inject() (
     } yield ok
   }
 
+  def failOverridePage(applicationId: ApplicationId) = loggedInThruStrideWithApplicationAndSubmission(applicationId) { implicit request =>
+    successful(Ok(termsOfUseFailOverridePage(ViewModel(applicationId, request.application.name))))
+  }
+
+  def failOverrideAction(applicationId: ApplicationId) = loggedInThruStrideWithApplicationAndSubmission(applicationId) { implicit request =>
+    request.body.asFormUrlEncoded.getOrElse(Map.empty).get("override").flatMap(_.headOption) match {
+      case Some("yes") => successful(Redirect(uk.gov.hmrc.apigatekeeperapprovalsfrontend.controllers.routes.TermsOfUseFailedJourneyController.overrideApproverPage(applicationId)))
+      case _           => successful(Redirect(uk.gov.hmrc.apigatekeeperapprovalsfrontend.controllers.routes.TermsOfUseFailedJourneyController.listPage(applicationId)))
+    }
+  }
+
+  def overrideApproverPage(applicationId: ApplicationId) = loggedInThruStrideWithApplicationAndSubmission(applicationId) { implicit request =>
+    successful(Ok(termsOfUseOverrideApproverPage(approverForm, ViewModel(applicationId, request.application.name))))
+  }
+
+  def overrideApproverAction(applicationId: ApplicationId) = loggedInThruStrideWithApplicationAndSubmission(applicationId) { implicit request =>
+    def handleValidForm(form: ApproverForm) = {
+      submissionReviewService.updateEscalatedTo(form.firstName + " " + form.lastName)(request.submission.id, request.submission.latestInstance.index).map {
+        case Some(value) => Redirect(uk.gov.hmrc.apigatekeeperapprovalsfrontend.controllers.routes.TermsOfUseFailedJourneyController.overrideNotesPage(applicationId))
+        case None        => {
+          logger.warn("Persisting override escalated to failed")
```

Comment:
This log feels vague - could it be altered to have more context when investigating, e.g. applicationId?  Similarly for other logs added in this PR.

Reply:
Fixed